src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeoAuthProvider.scala

```diff
@@ -9,9 +9,8 @@ import scala.concurrent.Future
 
 object ProjectActions {
   sealed trait ProjectAction extends Product with Serializable
-  case object ListClusters extends ProjectAction
   case object CreateClusters extends ProjectAction
-  val allActions = Seq(ListClusters, CreateClusters)
+  val allActions = Seq(CreateClusters)
```

Comment:
Note that `ListClusters` is gone entirely. It's been replaced by the explicit function call `canSeeAllClustersInProject`, which Leo will try calling before falling back to making individual calls for each cluster.